shared/src/packages/orders/libs/constants/order-statuses.ts

```diff
@@ -0,0 +1,8 @@
+import { OrderStatus } from '../enums/order-status.enum.js';
+
+const ORDER_STATUSES = Object.values(OrderStatus as Record<number, string>) as [
+  string,
+  ...string[],
+];
```

Comment:
```suggestion
const ORDER_STATUSES = Object.values(OrderStatus) as [
  ValueOf<typeof OrderStatus>,
];
```

This should keep it typed yet compatible with pgEnum argument